sources/Common/src/Options.cpp

```diff
@@ -110,10 +113,17 @@ Options::parse(int argc, char* argv[]) -> std::tuple<bool, Request> {
     if (vm.count("log-level") > 0) {
       config_.dynawoLogLevel = vm["log-level"].as<ParsedLogLevel>().logLevelDefinition;
     }
-    return std::forward_as_tuple(true, Request::RUN_SIMULATION);
+
+    if (vm.count("flat") > 0) {
+      return Request::RUN_SIMULATION_NSA;
+    } else if (vm.count("contingencies") > 0) {
+      return Request::RUN_SIMULATION_SA;
+    } else {
+      return Request::RUN_SIMULATION_N;
```

Comment:
This means that if the contingencies is not defined with the flat option the flow goes on, we should get an error